grapple/types/pages.py

```diff
@@ -284,10 +286,17 @@ def resolve_pages(self, info, **kwargs):
                 pages = pages.in_site(site)
 
             content_type = kwargs.pop("content_type", None)
-            if content_type:
-                app_label, model = content_type.strip().lower().split(".")
+            content_types = content_type.split(",") if content_type else None
+            if content_types:
+                app_labels = []
+                models = []
+                for content_type in content_types:
+                    app_label, model = content_type.strip().lower().split(".")
+                    app_labels.append(app_label)
+                    models.append(model)
                 pages = pages.filter(
-                    content_type__app_label=app_label, content_type__model=model
+                    content_type__app_label__in=app_labels,
+                    content_type__model__in=models,
```

Comment:
This doesn't feel right, we need to check pairs of app label/model, rather than two lists. suggest constructing a set of OR-ed `Q()` objects

Reply:
good catch.  Updated and rebased.